src/helix-ui/styles/base/lists.less

```diff
@@ -1,5 +1,34 @@
+ul, ol {
+  list-style-type: none;
+  padding: 0;
+  
+  li {
+    margin-left: 0.5rem;
+    padding-left: 1rem;
+    position: relative;
+    
+    &::before {
+      left: 0;
+      position: absolute;
+    }
+  }
+}
+
+ol {
+  counter-reset: orderedlist;
+  
+  li::before {
+    counter-increment: orderedlist;
+    content: counter(orderedlist) '.';
+  }
```

Comment:
is there a way to use `counter-increment` with Roman Numerals? If so then you could add block here to target and style `type[i]` attribute.

Reply:
This is a suboptimal solution. Based on @araiford's feedback for the expected output, w shouldn't need CSS counters.